CHANGELOG.rst

```diff
@@ -0,0 +1,26 @@
+CHANGELOG
+=========
+
+.. The text for the changelog is generated with ``npm run changelog``
+.. Then it is formatted and copied into this file.
+.. This is included by docs/changelog.rst
+
+
+Version v1.1.0
+---------------
+
+The major changes in this release were to send the client version with the ad request.
+In the future, we will begin warning users if their ad client is very out of date.
```

Comment:
Do we have a way to do this in the client? I wonder if we should add some kind of support for showing a warning from the API response, so we can actually alert the older clients in the next release.

Reply:
That's my plan but it is not implemented.